contacts.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <!-- Global and Theme styling -->
+        <link rel="stylesheet" href="/global.css">
```

Comment:
no need to add the forward slash if the file is in root directory, we can use href="global.css"